dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.40/V2_40_5__add_evenhook_table.sql

```diff
@@ -0,0 +1,32 @@
+
+-- Add eventhook table
+
+drop table if exists eventhook;
+
+create table eventhook (
```

Comment:
Use "create table if not exists" for (at least part) idempotence.

Reply:
Sure, was just following the other flyway scripts, updated now.